cmd/svcat/output/binding.go

```diff
@@ -65,15 +65,20 @@ func WriteBindingList(w io.Writer, bindings ...v1beta1.ServiceBinding) {
 // WriteBindingDetails prints details for a single binding.
 func WriteBindingDetails(w io.Writer, binding *v1beta1.ServiceBinding) {
 	t := NewDetailsTable(w)
-
 	t.AppendBulk([][]string{
 		{"Name:", binding.Name},
 		{"Namespace:", binding.Namespace},
 		{"Status:", getBindingStatusFull(binding.Status)},
 		{"Instance:", binding.Spec.ServiceInstanceRef.Name},
 	})
-
 	t.Render()
+
+	// If available, show the parameters as they were sent to the broker, otherwise show what was requested
+	if binding.Status.ExternalProperties != nil && binding.Status.ExternalProperties.Parameters != nil {
```

Comment:
Is it the case that you never want to see the Spec.Parameters if you have Status.ExternalProperties.Parameters?

it is one or the other?

Reply:
I wasn't sure what was a good design choice for all users, so I went with what I personally prefer. Don't show me the params twice, and I favor knowing what was actually sent to the broker, vs what is in the spec.

Suggestions welcome on how to either make that more clear, or if you have a different UX in mind! ✨

Reply:
@kibbles-n-bytes do you have an opinion?

Reply:
Hmm... I'm inclined to say you'd want to actually see both. :frowning_face: I think describing the resource should still be showing its real state, even if its broken.

Reply:
@n3wscott I like your suggestion the best! 

Since the data is always available, I vote we just diff and point out differences when they exist proactively, and otherwise have the user show these other fields explicitly with a flag when desired. What about this?

Happy Path (current PR)
```console
$ svcat describe instance puppy-kisses
Name: puppy-kisses
... TRUNCATED

Parameters:
  type: golden-retriever
  age: 3
```

Looking at all the things because I want to
```console
$ svcat describe instance puppy-kisses --external-properties
Name: puppy-kisses
... TRUNCATED

Parameters:
  type: golden-retriever
  age: 3

External Properties:
  Cluster Service Plan External Name: Goldy
  Cluster Service Plan External ID: abc123
  
  Parameters:
    type: golden-retriever
    age: 3

  Parameter Checksum: abc1232
```

svcat detects when things are different and automatically shows me a diff
```console
$ svcat describe instance puppy-kisses
Name: puppy-kisses
... TRUNCATED

Parameters:
  type: golden-retriever
  age: 3

The following differences between the spec and the status have been identified:
  Paramters:
    age-spec:   3
    age-status: 
```

Reply:
Yeah, I think that would work best! I think I have a slight preference over formatting the "diff" section more as the following:

```
The following differences between the spec and the status have been identified:
  Spec:
    Parameters:
      type: corgi
      age: 5
    
  Status:
    Parameters:
      type: golden-retriever
      age: 3
```

Unless there's a better way to diff right next to each other in the yaml without having to rename the key, since there's a potential for the key to clash or be unclear since JSON allows any Unicode character.

Reply:
I just made up a diff format for my example. 😀  I am pretty sure there are existing diff libraries out there that we could reuse, or if we don't like any of the diff formats, I like your idea over writing our own diff-magic. 👍